tests/render/test_render_BulletListContentBlock.py

```diff
@@ -66,78 +54,3 @@ def test_parse_row_condition_string_pandas_engine():
             "row_condition__3": "PClass != '1st'",
         },
     )
-
-
-@pytest.mark.filesystem
-def test_all_expectations_using_test_definitions():
```

Comment:
Maybe I should restore this test? I still need to look at this closer. It uses all the json test_definition files.